GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -0,0 +1,61 @@
+﻿using CommandLine;
+using GVFS.Common;
+using System;
+
+namespace GVFS.CommandLine
+{
+    [Verb(ConfigVerbName, HelpText = "Set and Get GVFS config settings.")]

```

Comment:
"Get" should be lowercase

Reply:
Also reword -- "Get and set GVFS options"